content/en/real_user_monitoring/browser/modifying_data_and_context.md

```diff
@@ -82,74 +82,223 @@ window.DD_RUM &&
-```javascript
-window.DD_RUM.onReady(function() {
-    window.DD_RUM.startView({
-      name: 'checkout',
-      service: 'purchase',
-      version: '1.2.3'
-    })
-})
-```
-{{% /tab %}}
-{{% tab "CDN sync" %}}
-```javascript
-window.DD_RUM && window.DD_RUM.startView({
-  name: 'checkout',
-  service: 'purchase',
-  version: '1.2.3'
-})
-```
-{{% /tab %}}
-{{< /tabs >}}
-</details>
-
-<details>
-  <summary>before <code>v4.13.0</code></summary>
-The following example manually tracks the page views on the <code>checkout</code> page in a RUM application. No service or version can be specified.
-
-{{< tabs >}}
-{{% tab "NPM" %}}
-```javascript
-datadogRum.startView('checkout')
-```
-
-{{% /tab %}}
-{{% tab "CDN async" %}}
-```javascript
-window.DD_RUM.onReady(function() {
-    window.DD_RUM.startView('checkout')
-})
-```
-{{% /tab %}}
-{{% tab "CDN sync" %}}
-```javascript
-window.DD_RUM && window.DD_RUM.startView('checkout')
-```
-{{% /tab %}}
-{{< /tabs >}}
-
-</details>
-
+   <details open>
+     <summary>Latest version</summary>
+   The following example manually tracks the page views on the <code>checkout</code> page in a RUM application. Use <code>checkout</code> for the view name and associate the <code>purchase</code> service with version <code>1.2.3</code>.
+
+   {{< tabs >}}
+   {{% tab "NPM" %}}
+   ```javascript
+   datadogRum.startView({
+     name: 'checkout',
+     service: 'purchase',
+     version: '1.2.3'
+   })
+   ```
+
+   {{% /tab %}}
+   {{% tab "CDN async" %}}
+   ```javascript
+   window.DD_RUM.onReady(function() {
+       window.DD_RUM.startView({
+         name: 'checkout',
+         service: 'purchase',
+         version: '1.2.3'
+       })
+   })
+   ```
+   {{% /tab %}}
+   {{% tab "CDN sync" %}}
+   ```javascript
+   window.DD_RUM && window.DD_RUM.startView({
+     name: 'checkout',
+     service: 'purchase',
+     version: '1.2.3'
+   })
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}
+   </details>
+
+   <details>
+     <summary>before <code>v4.13.0</code></summary>
+   The following example manually tracks the page views on the <code>checkout</code> page in a RUM application. No service or version can be specified.
+
+   {{< tabs >}}
+   {{% tab "NPM" %}}
+   ```javascript
+   datadogRum.startView('checkout')
+   ```
+
+   {{% /tab %}}
+   {{% tab "CDN async" %}}
+   ```javascript
+   window.DD_RUM.onReady(function() {
+       window.DD_RUM.startView('checkout')
+   })
+   ```
+   {{% /tab %}}
+   {{% tab "CDN sync" %}}
+   ```javascript
+   window.DD_RUM && window.DD_RUM.startView('checkout')
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}
+
+   </details>
 
 If you are using React, Angular, Vue, or any other frontend framework, Datadog recommends implementing the `startView` logic at the framework router level.
 
+### React Native router instrumentation
+
+To override default RUM view names so that they are aligned with how you've defined them in your React Native application, you need to follow the below steps.
```

Comment:
similar to https://github.com/DataDog/documentation/pull/18760#discussion_r1250474297
```suggestion
To override default RUM view names so that they are aligned with how you've defined them in your React application, you need to follow the below steps.
```